plugin/cypress/e2e/sidebar-navigation.ts

```diff
@@ -1,67 +1,94 @@
-import { Given, Then, When } from "@badeball/cypress-cucumber-preprocessor";
+import { Before, Then, When } from '@badeball/cypress-cucumber-preprocessor';
+import { getApiProxy } from '../support/utils';
+
+Before(() => {
+  // This prevents cypress from stopping on errors unrelated to the tests.
+  // There can be random failures due timeouts/loadtime/framework that throw browser errors.  This
+  // prevents a CI failure due something like a "slow".  There may be a better way to handle this.
+  cy.on('uncaught:exception', (err, _runnable, promise) => {
+    // when the exception originated from an unhandled promise
+    // rejection, the promise is provided as a third argument
+    // you can turn off failing the test in this case
+    if (promise || err.message.includes('MobX')) {
```

Comment:
I saw the MobX error messages on failed installation of OSSMC, but was unable to reproduce so far